pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -886,40 +888,42 @@ void SYSCALL()
 					const char* field = (cpuRegs.GPR.n.a2.UL[0] & 1) ? "FRAME" : "FIELD";
 					std::string mode;
 					// Warning info might be incorrect!
-					switch (cpuRegs.GPR.n.a1.UC[0]) {
-						case 0x2:  mode = "NTSC 640x448 @ 59.940 (59.82)"; break;
-
-						case 0x3:  mode = "PAL  640x512 @ 50.000 (49.76)"; break;
-
-						case 0x1A: mode = "VESA 640x480 @ 59.940"; break;
-						case 0x1B: mode = "VESA 640x480 @ 72.809"; break;
-						case 0x1C: mode = "VESA 640x480 @ 75.000"; break;
-						case 0x1D: mode = "VESA 640x480 @ 85.008"; break;
-
-						case 0x2A: mode = "VESA 800x600 @ 56.250"; break;
-						case 0x2B: mode = "VESA 800x600 @ 60.317"; break;
-						case 0x2C: mode = "VESA 800x600 @ 72.188"; break;
-						case 0x2D: mode = "VESA 800x600 @ 75.000"; break;
-						case 0x2E: mode = "VESA 800x600 @ 85.061"; break;
-
-						case 0x3B: mode = "VESA 1024x768 @ 60.004"; break;
-						case 0x3C: mode = "VESA 1024x768 @ 70.069"; break;
-						case 0x3D: mode = "VESA 1024x768 @ 75.029"; break;
-						case 0x3E: mode = "VESA 1024x768 @ 84.997"; break;
-
-						case 0x4A: mode = "VESA 1280x1024 @ 63.981"; break;
-						case 0x4B: mode = "VESA 1280x1024 @ 79.976"; break;
-
-						case 0x50: mode = "HDTV   720x480 @ 59.94"; break;
-						case 0x51: mode = "HDTV 1920x1080 @ 60.00"; break;
-						case 0x52: mode = "HDTV  1280x720 @ ??.???"; break;
-						case 0x53: mode = "HDTV   768x576 @ ??.???"; break;
-						case 0x54: mode = "HDTV 1920x1080 @ ??.???"; break;
-
-						case 0x72: mode = "DVD NTSC 640x448 @ ??.???"; break;
-						case 0x73: mode = "DVD PAL/480P 720x480 @ ??.???"; break;
-
-						default: Console.Error("Mode %x is not supported. Report me upstream", cpuRegs.GPR.n.a1.UC[0]);
+					switch (cpuRegs.GPR.n.a1.UC[0])
+					{
+						case 0x0: case 0x2:  mode = "NTSC 640x448 @ 59.940 (59.82)"; gsSetVideoMode(GS_VideoMode::NTSC); break;
```

Comment:
it is better to put the case on 2 lines. Because it is hard to see there are 2 cases by line.

(And same below).